content/ngf/overview/gateway-architecture.md

```diff
@@ -52,205 +37,141 @@ Key functionalities include:
+    style GatewayB fill:#66CDAA,stroke:#333,stroke-width:2px
     style NGFPod fill:#66CDAA,stroke:#333,stroke-width:2px
-
-    style NGINXProcessAB fill:#66CDAA,stroke:#333,stroke-width:2px
-    style NGINXProcessC fill:#66CDAA,stroke:#333,stroke-width:2px
-
+    style NGINXProcessA fill:#66CDAA,stroke:#333,stroke-width:2px
+    style NGINXProcessB fill:#66CDAA,stroke:#333,stroke-width:2px
     style KubernetesAPI fill:#9370DB,stroke:#333,stroke-width:2px
-
-    style HTTPRouteAAndApplicationA fill:#E0FFFF,stroke:#333,stroke-width:2px
-    style HTTPRouteBAndApplicationB fill:#E0FFFF,stroke:#333,stroke-width:2px
-
-    style AppDevA fill:#FFA07A,stroke:#333,stroke-width:2px
+    style HTTPRouteAAndApplications fill:#E0FFFF,stroke:#333,stroke-width:2px
+    style HTTPRouteBandApplications fill:#E0FFFF,stroke:#333,stroke-width:2px
+    style UserDevA fill:#FFA07A,stroke:#333,stroke-width:2px
     style HTTPRouteA fill:#FFA07A,stroke:#333,stroke-width:2px
+    style HTTPRouteB fill:#FFA07A,stroke:#333,stroke-width:2px
     style ApplicationA fill:#FFA07A,stroke:#333,stroke-width:2px
-    style ClientsA fill:#FFA07A,stroke:#333,stroke-width:2px
-
-    style AppDevB fill:#87CEEB,stroke:#333,stroke-width:2px
-    style HTTPRouteB fill:#87CEEB,stroke:#333,stroke-width:2px
-    style ApplicationB fill:#87CEEB,stroke:#333,stroke-width:2px
-    style ClientsB fill:#87CEEB,stroke:#333,stroke-width:2px
+    style ApplicationB fill:#FFA07A,stroke:#333,stroke-width:2px
+    style ClientA fill:#FFA07A,stroke:#333,stroke-width:2px
+    style UserDevB fill:#87CEEB,stroke:#333,stroke-width:2px
+    style HTTPRouteC fill:#87CEEB,stroke:#333,stroke-width:2px
+    style ApplicationC fill:#87CEEB,stroke:#333,stroke-width:2px
+    style ClientB fill:#87CEEB,stroke:#333,stroke-width:2px
+    style PublicEndpoint fill:#FFD700,stroke:#333,stroke-width:2px
+```
 
-    style AppDevC fill:#FFC0CB,stroke:#333,stroke-width:2px
-    style HTTPRouteC fill:#FFC0CB,stroke:#333,stroke-width:2px
-    style ApplicationC fill:#FFC0CB,stroke:#333,stroke-width:2px
-    style ClientsC fill:#FFC0CB,stroke:#333,stroke-width:2px
+{{< call-out "note" >}} The figure does not show many of the necessary Kubernetes resources the Cluster Operators and Application Developers need to create, like deployment and services. {{< /call-out >}}
 
-    style PublicEndpointAB fill:#FFD700,stroke:#333,stroke-width:2px
-    style PublicEndpointC fill:#FFD700,stroke:#333,stroke-width:2px
+The figure shows:
 
-    %% Styling
-    classDef dashedSubgraph stroke-dasharray: 5, 5;
+{{< bootstrap-table "table table-bordered table-striped table-responsive" >}}
 
-    %% Assign Custom Style Classes
-    class DataplaneComponentsAB dashedSubgraph;
-    class DataplaneComponentsC dashedSubgraph;
-```
+| **Category**           | **Description**                                                                                                                                       |
```

Comment:
We technically should not be using the `NGF` acronym in our docs, so let's replace that anywhere with NGINX Gateway Fabric.